mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -402,6 +309,67 @@ bool GlAnimationOverlayCalculator::LoadAnimationAndroid(
       LOG(ERROR) << "Failed to read indices for frame " << frame_count_;
       return false;
     }
+
+    // Set triangle_mesh normals for shader usage
+    int len = lengths[2];
+    triangle_mesh.normals.reset(new float[len]);
+    float triangle_normals[len];
+
+    // Compute every triangle surface normal and store them for later use.
+    for (int idx = 0; idx < len; idx += 3) {
+      int v1 = triangle_mesh.triangle_indices.get()[idx];
+      int v2 = triangle_mesh.triangle_indices.get()[idx + 1];
+      int v3 = triangle_mesh.triangle_indices.get()[idx + 2];
+      // Previous (V1) vertex X,Y,Z indices in triangle_mesh.vertices
```

Comment:
super-nit: These are just 3 (ordered) vertices, there's not really any notion of previous/current/next.  The calculations here look good though.